qemu/tests/cfg/usb_device_hotplug.cfg

```diff
@@ -0,0 +1,75 @@
+- usb_device_hotplug:
+    virt_test_type = qemu
+    no Host_RHEL.m6.u1
+    restart_vm = yes
+    kill_vm_on_error = yes
+    usbs = " usbtest"
+    usb_bus = "usbtest.0"
+    type = usb_device_hotplug
```

Comment:
do we have other use device hotplug case? if yes, suggest to use use_storage_hotplug for this case